src/helpers/index.js

```diff
@@ -381,6 +381,12 @@ exports.xmlCode = object => {
   return Prism.highlight(object, Prism.languages.xml, 'xml')
 }
 
+exports.getDefaultTemplateData = () => {
+  return {
```

Comment:
I thought i was going to need more things in here so i made this helper. Maybe other things belong here and it's worth keeping? Or i can just delete this and move this NODE_ENV template var back into the router